tools/builder/CMakeLists.txt

```diff
@@ -3,40 +3,47 @@ set (SC_BUILDER_DIR "${SC_TOOLS_DIR}/builder")
 file(MAKE_DIRECTORY "${SC_BUILDER_DIR}/src/parser")
 
 
-add_custom_command(OUTPUT "${SC_BUILDER_DIR}/src/parser/scsLexer.c" 
-						"${SC_BUILDER_DIR}/src/parser/scsLexer.h" 
-						"${SC_BUILDER_DIR}/src/parser/scsParser.c" 
-						"${SC_BUILDER_DIR}/src/parser/scsParser.h"
+add_custom_command(OUTPUT "${SC_BUILDER_DIR}/src/parser/scsLexer.c"
```

Comment:
Project's cmake files should have the same code styles.
Revert the previous code styles and leave only build fixes.